ocaml/lambda/translcore.ml

```diff
@@ -1133,7 +1147,8 @@ and transl_apply ~scopes
             Lvar _ | Lconst _ -> (lam, layout)
           | _ ->
               let id = Ident.create_local name in
-              defs := (id, layout, lam) :: !defs;
+              let uid = Uid.internal_not_actually_unique in
+              defs := (id, uid, layout, lam) :: !defs;
```

Comment:
Seems like we don't really need to add `uid` to this tuple of they are all going to be `internal_not_actually_unique`.